README.md

```diff
@@ -44,8 +43,8 @@ Open resulting html file in favorite browser.
 
 ## Usage
 1. If you haven't `yarn install`
-2. `yarn build`
-3. `./bin/lodestar --help`
+2. `lerna run build`
```

Comment:
Don't we want to run bootstrap as well?

Reply:
when using yarn and workspaces, yarn install will bootstrap automatically, try it out :)